plinth/modules/minetest/__init__.py

```diff
@@ -51,6 +51,10 @@
           'is needed.'), box_name=_(cfg.box_name)),
 ]
 
+description_frontpage = [
+    _('<a href=\"apps/minetest\">Configure</a>'),
```

Comment:
Notice how each description_frontpage has some things in common:
* Starts with '<a href=\"apps/'.
* Ends with '\">Configure</a>'.
* Although it's a list, each one only has one element.
So I suggest that the HTML tag and "Configure" text should be moved to the index template. The only variable here is the URL.